packages/shared-types/src/lib/callback.types.ts

```diff
@@ -0,0 +1,43 @@
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+const callbackType = {
```

Comment:
Ported for compatibility.

Reply:
ignoring the unused variable because we arent using the `callbackType` object anywhere.

Reply:
Actually now that I think of this, this would technically be breaking....should we go back to enum?